advanced/yeoman-generator.md

```diff
@@ -32,43 +32,38 @@ You will then be guided through these questions:
 1. **Plugin name?**  
 The name of the plugin.
 2. **How do you want to write your plugin?**  
-With what technology do you want to write your plugin? ES2015 transpiled with Babel, Typescript or Bolt (which is the module system used internally by Tiny, and probably not something that will be that interesting for external developers to use).
+With what technology do you want to write your plugin? ES2015 transpiled with Babel, Typescript or the module system used internally by Tiny called Bolt.
 3. **Use yarn instead of npm?**  
-If you have yarn installed you can use that instead of npm for a speedier install.
+Use yarn instead of npm for a speedier install.
 4. **Skip git repo initialization?**  
-Most developers probably want to create a new repository for the plugin, but if you want to skip it this is the place to do it.
+Here you can skip the creation of a new repository for the plugin.
 5. **What’s your name?**  
 For license.
 6. **Your email? (optional)**  
 For license.
 7. **Your website? (optional)**  
 For license
 8. **Which license do you want to use?**  
-Hopefully self-explanatory: decide under what license you want to release you plugin.
+Choose the license for the plugin.
 
-After you have made your way through these prompts yeoman will install the needed dependencies and we are pretty much done: the project is bootstrapped and ready. `cd` into the plugin directory and run the following to start the auto reloading development server and get started:
+Yeoman installs the needed dependencies and the project is bootstrapped and ready. `cd` into the plugin directory and run the following command to start the auto reloading development server:
```

Comment:
Change "auto reloading" to "auto-reloading".

Reply:
done